safe_transaction_service/history/tests/test_views.py

```diff
@@ -865,7 +879,7 @@ def test_get_multisig_transactions(self):
         self.assertEqual(response.data["count"], 0)
         self.assertEqual(response.data["count_unique_nonce"], 0)
 
-        multisig_tx = MultisigTransactionFactory(safe=safe_address)
+        multisig_tx = MultisigTransactionFactory(safe=safe_address, proposer=proposer)
```

Comment:
How is this working if you didn't update `MultisigTransactionFactory`?

Reply:
because proposer can be null, we haven't data about who was the proposer on old values.